config/locales/client.en.yml

```diff
@@ -1444,6 +1454,11 @@ en:
       second_factor_backup_title: "Two Factor Backup"
       second_factor_backup_description: "Please enter one of your backup codes:"
       second_factor: "<a href>Log in using Authenticator app</a>"
+      security_key_description: "When you have your physical security key prepared press the Authenticate with Security Key button below."
+      security_key_alternative: "<a href>Can't find your security key or want to use another method?</a>"
```

Comment:
This is a bit odd - why is there an empty link in the translation? It would be better to keep such markup in the handlebars code.

Reply:
The above `second_factor` translation has it as well, I thought that it must have been a convention followed. I will change it and the other translation to not contain an empty link